src/main/java/Duke.java

```diff
@@ -1,10 +1,112 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
+    private final static String welcome_default = " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
+    private final static String line = "\t----------------------------------------------------\n";
+    private final static String welcome_changed = line + "\t" + "Hewwo fweind, I am fuwwy, your personal assitant,\n" +
+            "\t" + "How can I help you?\n" + line;
+    private static ArrayList<Task> tasklist = new ArrayList<>();
+
+    private static void fuwwyEcho(String echo) {
+        System.out.println(line + "\t" + echo + "\n" + line);
+    }
+
+    private static void addTask(Task task) {
+        tasklist.add(task);
+        fuwwyEcho("Uwu added:\t"
+                + task);
+    }
+
+    private static void deleteTask(int num) {
+        int id = num - 1;
+        Task delete = tasklist.get(id);
+        tasklist.remove(id);
+        fuwwyEcho("Ono, this task got yeeted:\n\t" + delete + "\n\tNow you have "
+                + tasklist.size() + " tasks in the list, Yay!\n");
+    }
+
+    private static void taskDone(int n) {
+        Task t = tasklist.get(n - 1);
+        t.setDone(true);
+        fuwwyEcho("Uwu! I've marked this task as done:\n\t" + t + "\n");
+    }
+
+    private static void eorD(String command, String task) throws DukeException {
```

Comment:
Method name can once again be more accurately descriptive. camelCase capitalisation should also be followed.